transformations/mr_value_replacement/transformation.py

```diff
@@ -0,0 +1,88 @@
+import torchtext.vocab
+import torch
+from torchtext.data import get_tokenizer
+from interfaces.KeyValuePairsOperation import KeyValuePairsOperation
+import spacy
+from itertools import product
+import random
+from random import sample
+from tasks.TaskTypes import TaskType
+
+class MRValueReplacement(KeyValuePairsOperation):
+    tasks = [TaskType.E2E_TASK]
+    languages = ["en"]
+  
+    def __init__(
+        self, seed=0, n_similar=10, max_outputs=1):
+        super().__init__(seed, max_outputs=max_outputs)
+
+        random.seed(seed)
+        self.glove = torchtext.vocab.GloVe(name = "6B", dim = "100")
+        self.spacy_model = spacy.load('en_core_web_sm')
```

Comment:
You might want to use spacy like [this](https://github.com/GEM-benchmark/NL-Augmenter/blob/d39f0e5f57a79937296351996617ac03ee80fb74/transformations/close_homophones_swap/transformation.py#L55).

Reply:
Seconded.

Reply:
Done!